vite.config.ts

```diff
@@ -2,17 +2,26 @@ import { defineConfig } from '@ownclouders/extension-sdk'
 import { readFileSync } from 'fs'
 import { join } from 'path'
 
-const certsDir = `${__dirname}/dev/docker/traefik/certificates`
+const isProduction = process.env.PRODUCTION
 
-export default defineConfig({
-  server: {
+let server
```

Comment:
```suggestion
let server = false
```

Reply:
Can the initialization be done in boolean and later reinitialized asa object with build and server config? My IDE doesn't allow this. Thus I feel like empty initialization make more sense

Reply:
yes, js is dynamic. but if ide complains, `if-else` is fine